neat_ml/tests/test_detection.py

```diff
@@ -0,0 +1,76 @@
+from pathlib import Path
+import re
+
+import matplotlib
+import numpy as np
+import pandas as pd
+import cv2
+matplotlib.use("Agg")
+import pytest
+from matplotlib.testing.compare import compare_images
+import pooch  # type: ignore[import-untyped]
+
+from neat_ml.opencv.detection import (
+    collect_tiff_paths,
+    _detect_single_image,
+    _save_debug_overlay,
+    run_opencv,
+)
+
+def test_visual_regression_debug_overlay(
```

Comment:
This test is fairly slow -- the second slowest in the testsuite now, probably because `pooch` DL:
 from `python -m pytest -n 8 --durations=10`: 
- `4.74s call     neat_ml/tests/test_detection.py::test_visual_regression_debug_overlay`
- `3.50s setup    neat_ml/tests/test_detection.py::test_visual_regression_debug_overlay`